packages/table/src/cell/editableCell.tsx

```diff
@@ -168,6 +174,14 @@ export class EditableCell extends React.Component<IEditableCellProps, IEditableC
         }
     }
 
+    private handleKeyPress = () => {
+        if (this.state.isEditing) {
+            return;
+        }
+        // setting dirty value to empty string because apparently the text field will pick up the key and write it in there
```

Comment:
This is 123 characters long. Curious why `prettier` didn't catch this. Can we line-wrap comments to 80 for readability?

Also, can you elaborate a little more on this case?

Reply:
Not sure why it wasn't caught. Basically, apparently, you don't need to insert the character yourself. When switched over to an empty text input (or non-empty one, technically), the bubbling up to the top of the browser will apparently get it to write a character. This is... incredibly weird, but, clearly what is happening, and I'm much happier to let chrome type the character than hope that we got the right one out of the keyboard event.